source/frontend/src/features/admin/financial-codes/list/FinancialCodeListView.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button } from 'components/common/buttons/Button';
+import { TableSort } from 'components/Table/TableSort';
+import { Roles } from 'constants/roles';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import orderBy from 'lodash/orderBy';
+import { Api_FinancialCode } from 'models/api/FinancialCode';
+import moment from 'moment';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaPlus } from 'react-icons/fa';
+import { useHistory } from 'react-router';
+import styled from 'styled-components';
+
+import { useFinancialCodeRepository } from '../hooks/useFinancialCodeRepository';
+import FinancialCodeFilter, {
+  defaultFinancialCodeFilter,
+  IFinancialCodeFilter,
+} from './FinancialCodeFilter/FinancialCodeFilter';
+import { FinancialCodeResults } from './FinancialCodeResults/FinancialCodeResults';
+import * as Styled from './styles';
+
+/**
+ * Page that displays acquisition files information.
+ */
+export const FinancialCodeListView: React.FC = () => {
```

Comment:
test coverage on sorting/filtering could use some improvement.

Reply:
Will have to see if there's a pattern already for testing sorting and filtering on list views. I think what I've seen is that we test the sub-components (results and filter) but will take a look

Reply:
I checked across all existing list views and couldn't find tests covering sorting/filtering (at the list view level). I agree this area should be improved but given that there's no existing pattern to follow this should go into a tech-debt task to add coverage for all list views not just this one